genienlp/arguments.py

```diff
@@ -139,63 +130,20 @@ def parse_argv(parser):
     parser.add_argument("--num_beams", type=int, nargs='+', default=[1], help='1 disables beam seach')
     parser.add_argument("--no_repeat_ngram_size", type=int, nargs='+', default=[0], help='ngrams of this size cannot be repeated in the output. 0 disables it.')
 
-    parser.add_argument("--almond_has_multiple_programs", action='store_true', help='Indicate if almond dataset has multiple programs for each sentence')
+    parser.add_argument('--model', type=str, choices=['BertLSTM', 'Bart'], default='BertLSTM', help='which model to import')
+    parser.add_argument('--pretrained_model', default=None,
```

Comment:
I'd like to keep BertLSTM because that's the name we used in the papers.

Reply:
The SPL paper uses XLM-R-LSTM.

Reply:
Yeah, I would also prefer the general term since the encoder can literally be any model from hugging face now that specific tokenizers are no longer needed.